src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4695,11 +4695,10 @@ namespace VariableSpeedCoils {
                         SupTemp -= FanCoolLoad / (CpAir * rhoair * VolFlowRate);
                     }
                     MixWetBulb = Psychrometrics::PsyTwbFnTdbWPb(state, MixTemp, MixHumRat, state.dataEnvrn->StdBaroPress, RoutineName);
-                    // need to use OutTemp for air-cooled and RatedInletWaterTemp for water-cooled
-                    if (varSpeedCoil.CondenserInletNodeNum != 0) {
-                        RatedSourceTempCool = RatedInletWaterTemp;
-                    } else {
+                    if (varSpeedCoil.CondenserType == DataHeatBalance::RefrigCondenserType::Air) {
                         RatedSourceTempCool = OutTemp;
+                    } else {
+                        RatedSourceTempCool = GetVSCoilRatedSourceTemp(state, DXCoilNum, ErrorsFound);
                     }
                     TotCapTempModFac =
                         CurveManager::CurveValue(state, varSpeedCoil.MSCCapFTemp(varSpeedCoil.NormSpedLevel), MixWetBulb, RatedSourceTempCool);
```

Comment:
I just tried to pull the new function GetVSCoilRatedSourceTemp over to the generic sizing routines and realized that this is not correct. This is how it was before and this change simply followed that methodology. However, the TotCapTempModFac curve should use the actual peak conditions (i.e., outdoor T, inlet air WB, inlet water T) used to size the coil. The TotCapTempModFac result is then used to get to a rated coil size that will meet the load at those operating conditions (i.e., RatedCapCoolTotalDes = CoolCapAtPeak / TotCapTempModFac;). If the actual rating conditions are passed to the CapFT curve, the result should be close to 1 and there will be no adjustment to capacity.

Reply:
Agreed, I'm planning to address that in #9439 as per our previous discussion for single speed water-to-air coils. I can make the same change for VS coils too if need be. As you mentioned, in this PR I kept the methodology the same and only addressed the fact the the OAT was used in place a EWT.